app/Helpers/shortcode.php

```diff
@@ -59,86 +58,43 @@ function url2shortcode(string $url): string
     return $link;
 }
 
-function normalize_shortcodes(string $value): string
+function normalize_shortcodes(string $input): string
```

Comment:
Minor: I noticed that the preview path doesn't go through this code.

![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/d04ed3c0-51b9-42ba-b739-3e2a1aa1c854)

Reply:
_preview.php_ in latest has been adjusted so it also executes this function before calling the shortcode helper.